### PR TITLE
Adding reporting service and dashboard frontend

### PR DESCRIPTION
**Pull request by benjay10**
This PR aims to introduce the [reporting generation service](https://github.com/lblod/loket-report-generation-service) and the [dashboard frontend](https://github.com/lblod/loket-report-generation-service) to the stack.

### CODE REVIEW

File: config/authorization/config.ex [hunk 1]
```diff
@@ -79,6 +79,11 @@ defmodule Acl.UserGroups.Config do
                         "https://data.vlaanderen.be/ns/mobiliteit#Verkeersbordcategorie",
                         "https://data.vlaanderen.be/ns/mobiliteit#Verkeersbordconcept",
                         "https://data.vlaanderen.be/ns/mobiliteit#VerkeersbordconceptStatus",
+                        # For logs, reports (dashboard and report service)
+                        "http://persistence.uni-leipzig.org/nlp2rdf/ontologies/rlog#Entry",
```
Comment (nvdk) [inline config/authorization/config.ex:83]: hmm should these be publicly accessible?

File: config/resources/master-jobs-domain.lisp [hunk 2]
```diff
@@ -0,0 +1,68 @@
+(define-resource job ()
+  :class (s-prefix "cogs:Job")
```
Comment (nvdk) [inline config/resources/master-jobs-domain.lisp:2]: aren't these missing in the mu-auth config?

File: config/authorization/config.ex (hunk 1 above)
Comment (benjay10) [reply config/authorization/config.ex:83]: Just copying from loket here ([see here](https://github.com/lblod/app-digitaal-loket/blob/master/config/authorization/config.ex#L123)) and I'm not fluent in mu-auth config yet. It does mention as a TODO to not put a lot of that in public.
So what user role(s) should these entries be allocated to? Writer, publisher, signer, ... or should we create a new role? Do we make a new login mechanism for these users?

File: config/resources/master-jobs-domain.lisp (hunk 2 above)
Comment (benjay10) [reply config/resources/master-jobs-domain.lisp:2]: Yes, probably missing. The problem is that I can not test this yet without any data/reports. I simply don't know what else I am missing here.

**Comment by nvdk** [comment]
update lblod/frontend-dashboard:latest should now have the mu-login support . once confirmed it's working OK we can tag it, release it and bump to the actual version in this PR.

File: docker-compose.yml [hunk 3]
```diff
@@ -165,3 +165,52 @@ services:
...
+    labels:
+      - "logging=true"
+  dashboard-login:
+    image: semtech/mu-login-service:2.9.1
+    links:
+      - database:database
+    environment:
+      USERS_GRAPH: "http://mu.semte.ch/graphs/public"
+      SESSIONS_GRAPH: "http://mu.semte.ch/graphs/sessions"
+  dashboard-registration:
```
Comment (nvdk) [inline docker-compose.yml:204]: we don't want the registration service I think

**Pull request merged**